doc/using/platform-support.chapter.md

```diff
@@ -1,18 +1,49 @@
 # Platform Support {#chap-platform-support}
 
-Packages receive varying degrees of support, both in terms of maintainer attention and available computation resources for continuous integration (CI).
+Packages receive varying degrees of support, both in terms of maintainer attention and available computation resources for continuous integration (CI). We have 7 defined tiers denoting how well supported each platform is.
 
-Below is the list of the best supported platforms:
+## Tiers {#sec-platform-tiers}
 
-- `x86_64-linux`: Highest level of support.
-- `aarch64-linux`: Well supported, with most packages building successfully in CI.
-- `aarch64-darwin`: Receives better support than `x86_64-darwin`.
-- `x86_64-darwin`: Receives some support.
+### Tier 1 {#sec-platform-tier1}
 
-There are many other platforms with varying levels of support.
-The provisional platform list in [Appendix A] of [RFC046], while not up to date, can be used as guidance.
+[Tier 1](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-1) platforms receive the highest level of support where problems can block updates, platform-specific patches are freely applied, and most packages are expected to work.
 
-A more formal definition of the platform support tiers is provided in [RFC046], but has not been fully implemented yet.
+### Tier 2 {#sec-platform-tier2}
 
-[RFC046]: https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md
-[Appendix A]: https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#appendix-a-non-normative-description-of-platforms-in-november-2019
+[Tier 2](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-2) platforms are expected to remain functional with updates, receive platform-specific patches as needed, and have many packages built by Hydra with full ofBorg support.
+
+### Tier 3 {#sec-platform-tier3}
+
+[Tier 3](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-3) platforms may receive non-intrusive platform-specific fixes, have native bootstrap tools available with cross-build toolchains in binary cache, but updates might break builds on these platforms.
+
+### Tier 4-7 {#sec-platform-tier4-7}
+
+Platform Tiers [4 through 7](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-4) indicate varying levels of minimal support going from receiving only limited fixes to platforms with no support, but a path to support.
+
+## Breakdown {#sec-platform-breakdown}
+
+| Triple                                | Support Tier | Channel Blockers | Hydra Support | Ofborg Support | Bootstrap Tarballs | Cross Compiling Support |
```

Comment:
Might be nice to clarify whether "cross compiling support" means as a local system or cross system. Assuming the latter, perhaps "Cross Compilation Target"?

Reply:
I'd say it means `pkgsCross`, being that we can cross compile it.